nightfall-optimist/src/services/database.mjs

```diff
@@ -132,23 +131,23 @@ export async function getBlockByTransactionHashL1(transactionHashL1) {
   return db.collection(SUBMITTED_BLOCKS_COLLECTION).findOne(query);
 }
 
-export async function numberOfBlockWithTransactionHash(transactionHash) {
-  const connection = await mongo.connection(MONGO_URL);
-  const db = connection.db(OPTIMIST_DB);
-  const query = { transactionHashes: transactionHash };
-  return db.collection(SUBMITTED_BLOCKS_COLLECTION).countDocuments(query);
-}
-
-/**
-function to get a block by blockHash, if you know the hash of the block. This
-is useful for rolling back Timber.
-*/
-export async function getBlockByBlockHash(blockHash) {
-  const connection = await mongo.connection(MONGO_URL);
-  const db = connection.db(OPTIMIST_DB);
-  const query = { blockHash };
-  return db.collection(SUBMITTED_BLOCKS_COLLECTION).findOne(query);
-}
+// export async function numberOfBlockWithTransactionHash(transactionHash) {
```

Comment:
Can we delete this instead of commenting out?

Reply:
I removed a lot of functions but left this behind for later use